.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+  push:
+
+jobs:
+  e2e-kubernetes:
+    name: Kuebrnetes E2E tests
+    runs-on: ubuntu-18.04
+    env:
+      E2E_MINIKUBE_DRIVER: docker
+    strategy:
+      matrix:
+        k8s_version:
+          - v1.16.15
+          - v1.17.14
+          - v1.18.12
+        required: [ "required" ] # Key and value are arbitrary, these just look nice in the job names
+        include:
+          # Run on 1.19 but allow failure
+          - k8s_version: v1.19.4
+            required: "optional"
+    continue-on-error: ${{ matrix.required == 'optional' }}
```

Comment:
It seems like the UX in github for `continue-on-error` is not like `allow_failure` on travis/gitlab. Given that it actually counts as a failed job for the checks, I'll probably remove this.